anomalib/core/callbacks/normalization.py

```diff
@@ -2,12 +2,12 @@
 from typing import Any, Dict, Optional
 
 import pytorch_lightning as pl
-import torch
 from pytorch_lightning import Callback, Trainer
 from pytorch_lightning.utilities.types import STEP_OUTPUT
-from torch.distributions import LogNormal, Normal
+from torch.distributions import LogNormal
 
 from anomalib.models import get_model
+from anomalib.utils.normalization.cdf_normalization import normalize, standardize
```

Comment:
do we need to explicitly name the module as `cdf_normalization`. It's already located in `normalisation` package. I think we could rename it as just `cdf`. in this case, it would be like `from anomalib.utils.normalization.cdf import normalize, standardize`. It would also be slightly less verbose.